code/game/machinery/computer3/laptop.dm

```diff
@@ -63,8 +63,7 @@
 
 
 	AltClick()
-		if(Adjacent(usr))
-			open_computer()
+		open_computer()
```

Comment:
This `Adjacent()` check was a duplicate of the one already in the `open_computer()`-method ([laptop.dm#L41](https://github.com/inselc/Aurora.3/blob/c7211666314b9271b6be0d2d6e4bed4bbc289013/code/game/machinery/computer3/laptop.dm#L41)). Same goes for `close_computer()`.